winterfell/src/lib.rs

```diff
@@ -594,15 +594,14 @@
 #[cfg(test)]
 extern crate std;
 
-pub use air::{AuxRandElements, GkrVerifier};
+pub use air::AuxRandElements;
```

Comment:
Should we re-export `LogUpGkrEvaluator` from here?

Reply:
Done